### PR TITLE
Fix the SARIF report

### PR DESCRIPTION
**Pull request by KengoTODA**
To pass the [SARIF validator](https://github.com/microsoft/sarif-sdk/issues/1952#issuecomment-664038096). 

The latest SARIF file generated by SpotBugs is here: [spotbugs.sarif.txt](https://github.com/spotbugs/spotbugs/files/4988343/spotbugs.sarif.txt)

### CODE REVIEW

File: spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java [hunk 1]
```diff
@@ -39,4 +42,25 @@
      * Bugs were reported.
      */
     public static final int BUGS_FOUND_FLAG = 1;
+
+    public static int from(int errors, int missingClasses, int bugs) {
+        Logger logger = LoggerFactory.getLogger(ExitCodes.class);
```
Comment (ThrawnCA) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java:47]: Logger should probably be a class field, not local.

Comment (KengoTODA) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java:47]: The interface cannot have `private` field. Is it OK to make the field `static final`?

Comment (KengoTODA) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java:47]: change proposal:

```diff
diff --git a/spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java b/spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java
index 144f51898..0f27a2040 100644
--- a/spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java
+++ b/spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java
@@ -43,23 +43,24 @@ public interface ExitCodes {
      */
     public static final int BUGS_FOUND_FLAG = 1;
 
+    static final Logger LOGGER = LoggerFactory.getLogger(ExitCodes.class);
+
     public static int from(int errors, int missingClasses, int bugs) {
-        Logger logger = LoggerFactory.getLogger(ExitCodes.class);
         int exitCode = 0;
-        logger.info("Calculating exit code...");
+        LOGGER.info("Calculating exit code...");
         if (errors > 0) {
             exitCode |= ERROR_FLAG;
-            logger.debug("Setting 'errors encountered' flag ({})", ERROR_FLAG);
+            LOGGER.debug("Setting 'errors encountered' flag ({})", ERROR_FLAG);
         }
         if (missingClasses > 0) {
             exitCode |= MISSING_CLASS_FLAG;
-            logger.debug("Setting 'missing class' flag ({})", MISSING_CLASS_FLAG);
+            LOGGER.debug("Setting 'missing class' flag ({})", MISSING_CLASS_FLAG);
         }
         if (bugs > 0) {
             exitCode |= BUGS_FOUND_FLAG;
-            logger.debug("Setting 'bugs found' flag ({})", BUGS_FOUND_FLAG);
+            LOGGER.debug("Setting 'bugs found' flag ({})", BUGS_FOUND_FLAG);
         }
-        logger.debug("Exit code set to: {}", exitCode);
+        LOGGER.debug("Exit code set to: {}", exitCode);
 
         return exitCode;
     }
```

Comment (ThrawnCA) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java:47]: Wait, this is an interface? How is it providing a method implementation without using the `default` keyword?

Comment (KengoTODA) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/ExitCodes.java:47]: It's possible to add a static methods into interface.

Previously [I tried to add a new class named `ExitCode`](https://github.com/spotbugs/spotbugs/pull/1215/files), but it made code base confused (we have both `ExitCode` and `ExitCodes`). So this time I used static methods in the `ExitCodes` interface.

**Pull request merged**